src/main/java/com/clouway/chita/HttpClient.java

```diff
@@ -94,7 +94,11 @@
 
         }
 
-        inputStream = conn.getInputStream();
+        try {
```

Comment:
Why you are not using if statement. It will be faster then waiting to throw an exception.